modules/moa-accessing-your-cluster.adoc

```diff
@@ -10,31 +10,58 @@ To log in to your cluster, you must configure an Identity Provider (IDP).
 
 This procedure uses GitHub as an example IDP. 
 
+To view other supported IDPs, run `moactl create idp --help`.
+
 .Procedure
 
 . Add an IDP.
-.. Enter the following command to create an IDP backed by GitHub. Follow the prompts from the output to access your link:https://github.com/settings/developers[GitHub developer settings] and configure a new OAuth application.
+.. The following command creates an IDP backed by GitHub. After running the command, follow the interactive prompts from the output to access your link:https://github.com/settings/developers[Github developer settings] and configure a new OAuth application.
 +
 [source, terminal]
 ----
-$ moactl create idp --cluster rh-moa-test-cluster1 --type github                                                                                               
+$ moactl create idp --cluster=rh-moa-test-cluster --interactive 
 ----
 +
+.. Enter the following values:
++
+* Type of identity provider: github
```

Comment:
In these bullets, I'd suggest to put them all in backticks, since they're literal values that the user is entering: `github`, `organizations`, `rh-test-org`.